src/python_interpreter.rs

```diff
@@ -360,68 +385,71 @@ impl PythonInterpreter {
     ///
     /// Don't ask me why or how, this is just what setuptools uses so I'm also going to use
     pub fn get_tag(&self, manylinux: &Manylinux) -> String {
-        let platform = self.target.get_platform_tag(manylinux);
-
-        if self.target.is_unix() {
-            format!(
-                "cp{major}{minor}-cp{major}{minor}{abiflags}-{platform}",
-                major = self.major,
-                minor = self.minor,
-                abiflags = self.abiflags,
-                platform = platform
-            )
-        } else {
-            format!(
-                "cp{major}{minor}-none-{platform}",
-                major = self.major,
-                minor = self.minor,
-                platform = platform
-            )
+        match self.interpreter {
+            Interpreter::CPython => {
+                let platform = self.target.get_platform_tag(manylinux);
+                if self.target.is_unix() {
+                    format!(
+                        "cp{major}{minor}-cp{major}{minor}{abiflags}-{platform}",
+                        major = self.major,
+                        minor = self.minor,
+                        abiflags = self.abiflags,
+                        platform = platform
+                    )
+                } else {
+                    format!(
+                        "cp{major}{minor}-none-{platform}",
+                        major = self.major,
+                        minor = self.minor,
+                        platform = platform
+                    )
+                }
+            }
+            Interpreter::PyPy => {
+                // hack to never use manylinux for pypy
+                let platform = self.target.get_platform_tag(&Manylinux::Off);
+                // pypy uses its version as part of the ABI, e.g.
+                // pypy3 v7.1 => pp371-pypy3_71-linux_x86_64.whl
+                format!(
+                    "pp3{abi_tag}-pypy3_{abi_tag}-{platform}",
+                    abi_tag = self.abi_tag,
+                    platform = platform,
+                )
+            }
         }
     }
 
-    /// Generates the correct suffix for shared libraries
```

Comment:
Please re-add that doc comment. Probably it's the best to move the cpython logic to its own function and re-add the comment there

Reply:
Ok, I readded it and some on pypy here.